programming_course_description.txt

```diff
@@ -0,0 +1,8 @@
+

```

Comment:
The first line is empty. Is there a reason for that? If yes, maybe add an explanation (you could just write into the first line since the code ignores it). A user might change up your sentences in replicating the synthesis and make the mistake of writing into the first line otherwise.

Reply:
There is a reason indeed. The first line is ignored by the program that is why the text must start in the second line. We added a corresponding comment in the first line of the .txt file and an explanation in the readme.

Reply:
Good job, we think that makes it more clearer to potential users